index.js

```diff
@@ -0,0 +1,51 @@
+//const Books = [];

```

Comment:
- [ ] Following the best practices, you are not meant to leave unused code as a comment in your codebase. It makes your code look unprofessional. :wave: :smile: